### PR TITLE
Created the engage snap deltas page

### LINKED ISSUE #4834: snap deltas - landing page - build
[copy doc](https://docs.google.com/document/d/1BzilE80HtJ89mPXnhkaK0AYyYuPr8fnJPwnVUaMhJ9Y/edit)

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Created a new /engage/snap-deltas page
- Created a copy doc
- Tweaked the header include to have a width and a max-width and max-height

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/engage/snap-deltas](http://0.0.0.0:8001/engage/snap-deltas)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the [copy doc](https://docs.google.com/document/d/1BzilE80HtJ89mPXnhkaK0AYyYuPr8fnJPwnVUaMhJ9Y/edit#) and test the form

## Issue / Card

Fixes #4834

### CODE REVIEW

File: templates/engage/snap-deltas.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{% extends "engage/base_engage.html" %}
+
+{% block title %}Optimising IoT bandwidth with delta updates{% endblock %}
+
+{% block meta_description %}Learn how snap deltas' over-the-air updates are efficient and effective.{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1BzilE80HtJ89mPXnhkaK0AYyYuPr8fnJPwnVUaMhJ9Y/edit#t{% endblock meta_copydoc %}
+
+{% block content %}
+{% include "engage/shared/_header.html" with title="Optimising IoT bandwidth with delta updates"  image="25302166-snapcraft-primary-icon--dark.svg" subtitle="Learn how snap deltas' over-the-air updates are efficient and effective." url="#register-section" cta="Watch the webinar" class="p-strip--light" %}
```
Comment (caldav) [inline templates/engage/snap-deltas.html]:
```suggestion
{% include "engage/shared/_header.html" with title="Optimising IoT bandwidth with delta updates"  image="25302166-snapcraft-primary-icon--dark.svg" subtitle="Learn how snap deltas' over-the-air updates are efficient and effective." url="#register-section" cta="Read the whitepaper" class="p-strip--light" %}
```

File: templates/engage/snap-deltas.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+      <h4>Highlights of this whitepaper include:</h4>
+
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Overview of snaps' automatic update feature, their release cadence and how this helps manage software updates on IoT devices.</li>
+      <li class="p-list__item is-ticked">Explanation of snap deltas downloads and uploads on IoT devices, desktop and the cloud.</li>
+      <li class="p-list__item is-ticked">Case studies of deployed IoT devices from Dell, Rigado and Fingbox demonstrating the significant bandwidth savings of their respective snaps.</li>
+      </ul>
+    </div>
+    <div class="col-5">
```
Comment (Caleb-Ellis) [inline templates/engage/snap-deltas.html]: This should have the id for the CTA link

**Comment by anasereijo** [comment]
@pmahnke it's looking good although this needs to be changed once we have the takeover designed.

**Comment by anasereijo** [comment]
@pmahnke nice!

**Pull request merged**